Q5.py

```diff
@@ -0,0 +1,17 @@
+from itertools import permutations
+
+
+def list_sum(num_list, user_input):
+    #I have use itertools-library to get permutations of different element 
+    permutations_list = list(permutations(num_list, 2))
+    #i have separated the combination of list , who's sum is equal to user_input and convert into list
+    sum_list = [list(_) for _ in set(permutations_list) if sum(_) == user_input]
```

Comment:
Please change `_` everywhere you used. You should use this only when there is no use of `_` in the code

Reply:
Done Sir i have change it